docs/references/ic-interface-spec.md

```diff
@@ -2027,6 +2035,48 @@ When executing a query or composite query method via a query call (i.e. in non-r
 
     This traps if `ic0.data_certificate_present()` returns `0`.
 
+### Cycle cost calculation {#system-api-cycle-cost}
+
+Inter-canister calls have an implicit cost, and some calls to the management canister require the caller to attach cycles to the call explicitly.  
+The various cost factors may change over time, so the following system calls serve to give the canister programmatic, up-to-date information about the costs. 
+
+These system calls return costs in Cycles, represented by 128 bits, which will be written to the caller-specified `dst` pointer. 
+
+-   `ic0.cost_call : (method_name_size: i64, payload_size : i64, dst : I) -> () `I ∈ {i32, i64}`
+
+    The cost of an inter-canister call. `method_name_size` is the byte length of the method name, and `payload_size` is the length of the encoded argument to the method.  
```

Comment:
```suggestion
    The cost of an inter-canister call. `method_name_size` is the byte length of the method name, and `payload_size` is the byte length of the argument to the method. 
```
I'd drop "encoded" as it doesn't matter what the argument looks like (it can as well be plain text) and "byte length" should be unambiguous.